content/06.multiple-approaches-required.md

```diff
@@ -1,23 +1,33 @@
-#### Using composite approaches can be a powerful strategy
+### Using composite approaches can be a powerful strategy
 
 We have described multiple approaches for maximizing the success of machine learning applications in rare disease research throughout. 
 In practice, it is rarely sufficient to use one of these techniques in isolation. 
-Below, we highlight  two recent works – one supervised and one unsupervised – in the rare disease domain that draw on multiple concepts covered in the earlier sections. 
-Feature-representation-transfer, which incorporates dimension reduction through representation learning,  prior data, and regularization underlie both approaches.
+Below, we highlight two recent works in the rare disease domain that draw on multiple concepts covered in the earlier sections. 
+Feature-representation-transfer, which incorporates dimension reduction through representation learning, prior data, and regularization underlie both approaches.
 
-In a method termed DeepProfile, publicly available acute myeloid leukemia (AML) gene expression data was leveraged to improve the prediction of in-vitro drug responses [@doi:10.1101/278739]. 
-The authors trained a variational autoencoder (VAE), on AML data that had been collected over time without the desired phenotypic information (drug response). 
-The authors used the learned attributes to encode a low-dimensional representation of held-out AML data with drug response labels, and used this representation as input to a supervised classifier that predicted in vitro drug response, which outperformed using the original features. 
-The study by Dincer and colleagues highlights another challenge: samples collected as part of multiple studies may not be associated with the deep phenotypic information that would maximize their scientific value, but this does not preclude the use of these samples in learning representations.
+Thousands of acute myeloid leukemia (AML) patient gene expression samples have been collected over time and are publicly available.
+However, as is often the case with publicly available data, not all of these samples include the most relevant clinical or phenotypic data such as drug response.
+The authors of Dincer et al. had access to an _in vitro_ experiment that examined the response to 160 drugs for 30 AML patient samples, measured using genome-wide arrays which have tens of thousands of features [@doi:10.1038/s41467-017-02465-5].
+Training on rare disease datasets with a low number of samples, a considerable number of tasks, and many features can result in ML models that are of limited utility.
+The low number of samples in the drug response dataset could also be prohibitive when performing representation learning.
```

Comment:
If taking the above suggestion, this sentence would also need to be moved later in my opinion.